src/Data/List/Relation/Binary/Sublist/Heterogeneous/Properties.agda

```diff
@@ -470,6 +472,18 @@ decPoset ER-poset = record
   { isDecPartialOrder = isDecPartialOrder ER.isDecPartialOrder
   } where module ER = DecPoset ER-poset
 
+------------------------------------------------------------------------
+-- Reasoning over sublists
+------------------------------------------------------------------------
+
+module ⊆-Reasoning (≲ : Preorder a e r) where
+
+  open ≲-Reasoning (preorder ≲) public
+    renaming (≲-go to ⊆-go; ≈-go to ≋-go)
+
+  open ⊆-syntax _IsRelatedTo_ _IsRelatedTo_ ⊆-go public
+  open ≋-syntax _IsRelatedTo_ _IsRelatedTo_ ≋-go (Pw.symmetric (Preorder.Eq.sym ≲)) public
```

Comment:
There may be a nicer way of supplying these arguments by judicious `open`ing of the module parameter `≲`, but that's moot I think?

Reply:
I agree that my code is ugly. It is moot for the purposes of fixing up the one proof we used as a test case, but not moot for anyone wishing to use those combinators! In that case, all arguments really should be provided.